README.rst

```diff
@@ -66,29 +66,37 @@ Behind the scene, it uses `Ziggurat-Foundations`_ and `Authomatic`_.
 .. end-badges
 
 
-REST API Documentation
-======================
+Documentation
+=============
+
+The REST API documentation is auto-generated and served under ``{HOSTNAME}/api/`` using Swagger-UI with tag ``latest``.
```

Comment:
Swagger-UI with tag ``latest``, I know what it does, I am not familiar how it works.  Is it a python library or a javascript library?  

What are we pulling with this tag `latest`?  The library at docker image build time or the javscript library at client browser loading time directly from Swagger website?

Reply:
No. That's not related to a library or docker.

Within the swagger-ui interface available on `/api` path, there's a dropdown list of Magpie versions for old iterations of the API. The `latest` is the auto-generated one from the code.

Reply:
Ah I see.